myst_nbs/time_series/Air_passengers-Prophet_with_Bayesian_workflow.myst.md

```diff
@@ -151,7 +164,8 @@ with linear:
 fig, ax = plt.subplots(nrows=2, ncols=1, sharex=True)
```

Comment:
you can replace the `linear_posterior_predictive ...` line with: 

```
pm.sample_posterior_predictive(linear_trace, extend_inferencedata=True)
```

then use `linear_trace.posterior_predictive` (or `az.extract_dataset(linear_trace, group="posterior_predictive"...`)